coreneuron/permute/cellorder.cpp

```diff
@@ -600,15 +588,24 @@ void solve_interleaved2(int ith) {
     defined(_OPENACC)
         int nstride = stridedispl[nwarp];
 #endif
-        nrn_pragma_acc(parallel loop gang vector vector_length(
-            warpsize) present(nt [0:1],
+        /* If we compare this loop with the one from cellorder.cu (CUDA version), we will understand 
```

Comment:
@olupton @pramodk 
Here is the explanation on how the loop works in comparison to the CUDA counterpart.